source/apiVolontaria/apiVolontaria/views.py

```diff
@@ -115,6 +116,42 @@ def get_authenticators(self):
             self.authentication_classes = ()
         return [auth() for auth in self.authentication_classes]
 
+    def post(self, request, *args, **kwargs):
+        response = self.create(request, *args, **kwargs)
+        # get token for user save and send key this token with a mail
+        user = User.objects.get(username=request.data["username"])
+
+        if response.status_code == status.HTTP_201_CREATED:
+            try:
+                SERVICE_MAIL = settings.SETTINGS_IMAILING
+            except AttributeError:
```

Comment:
A comment should explain that the `AttributeError` happens if `SETTINGS_IMAILING` is not uncommented in `settings.py`